week-3/D-array-filter/exercise.js

```diff
@@ -8,8 +8,13 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
-
+var pairsByIndex = pairsByIndexRaw.filter(arrFiltering); // Complete this statement
+function arrFiltering(pair){
+  if (pair !== null && pair && pair !=="flase" && pair !== "whoops" && pair.length == 2){
```

Comment:
This works, but what if the input contains other arbitrary values like "hello" or "oops". Remember the "typeof" keyword can be handy in this situation!